projects/sound-open-firmware/fetch-sdk.sh

```diff
@@ -15,20 +15,13 @@
 #
 ################################################################################
 
-# Zephyr doesn't provide a "latest" link, so clone the source tree
-# that produces the SDK (much smaller than the tarball itself, so
-# minimal overhead) and find the latest version tag as a proxy.  Will
-# likely break if the script is run in the moments between tagging a
-# release and the tarball appearing on github, but the risk is low and
-# it will recover with a retry.
+# use the github API to fetch the latest stable SDK with assets
 
-git clone --filter=tree:0 https://github.com/zephyrproject-rtos/sdk-ng
+RELEASES=$(curl -L  -H "Accept: application/vnd.github+json"  -H "X-GitHub-Api-Version: 2022-11-28"  https://api.github.com/repos/zephyrproject-rtos/sdk-ng/releases)
+SDK=$(jq '[.[] | select(.prerelease == false) | {tag: .tag_name, assets: .assets[] | select(.name | contains("linux-x86_64_minimal.tar.xz"))}] | .[0]' <<< $RELEASES)
```

Comment:
This is much cleaner than the original for sure.  My facility with github API hacking and jq are both at the "well, I know those tools exist" level.